boards/silabs/radio_boards/xg24_rb4187c/xg24_rb4187c.yaml

```diff
@@ -9,13 +9,12 @@ toolchain:
   - gnuarmemb
   - xtools
 supported:
+  - bluetooth
   - gpio
   - uart
   - watchdog
 testing:
   ignore_tags:
-    - net
-    - bluetooth
     - pm
     - hwinfo
```

Comment:
@mrfuchs it seems you provided the initial board definition in commit 4310853d076e. Is there any particular reason you had all these features listed under `ignore_tags` or was it e.g. just a copy-paste from somewhere else?